opensubtitles/common.go

```diff
@@ -0,0 +1,143 @@
+package opensubtitles
+
+import "time"
+
+type Subtitles struct {
+	TotalPages int    `json:"total_pages"`
+	TotalCount int    `json:"total_count"`
+	Page       int    `json:"page"`
+	Data       []Data `json:"data"`
+}
+type Uploader struct {
```

Comment:
Codacy found an issue: [exported type Uploader should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)